src/ActionModal/components/ModalDeposit.vue

```diff
@@ -7,6 +7,7 @@
     title="Deposit"
     class="modal-deposit"
     submission-error-prefix="Depositing failed"
+    transaction-type="DepositTx"
```

Comment:
use transactionType enum from `transactionTypes.js`

Reply:
I wouldn't know then the transaction type 🤷‍♀ I need that prop to get the right gas Estimate.

Reply:
Yes, you should use the enum value. Like a variable. Instead of hardcoding the string value in here

Reply:
Oops, transactionTypes are the "old" values, MsgSubmitProposal and so on... I need to make a dictionary to convert them

Reply:
Well, actually it would be just much easier to change the values in `network-fees` and use the normal Cosmos-SDK messages. Or hardcode them. Dictionary feels like it is adding too much if it is not already there

Reply:
the correct types are in `src\components\transactions\messageTypes.js`. Use those and remove `transactionTypes.js` (will cause some refactoring)

Reply:
checking this rn

Reply:
ok doing it in another PR